obs-package-status/status-check.rb

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/ruby
+require 'net/http'
+require 'nokogiri'
+require 'json'
+# require 'pry'
+require 'dotenv/load'
+
+# OBS Configuration
+OBS_API_URL = ENV['OBS_API_URL'] || 'https://api.opensuse.org'
```

Comment:
I like using this form for default values: `ENV.fetch('OBS_API_URL', 'https://api.opensuse.org')`